src/edi/workflows.py

```diff
@@ -40,18 +57,24 @@ def __init__(self, input_message: Any):
         Configures the EdiProcess instance.
         Attributes include:
         - input_message: cached source message
+        - data_model: edi domain model (FHIR, HL7, X12, etc)
         - meta_data: EdiMessageMetadata object
         - metrics: EdiProcessingMetrics object
         - operations: List of EdiOperations completed for this instance
         """
 
         self.input_message = input_message
+        self.data_model = None
         self.meta_data: Optional[EdiMessageMetadata] = None
         self.metrics: EdiProcessingMetrics = EdiProcessingMetrics(
             analyzeTime=0.0, enrichTime=0.0, validateTime=0.0, translateTime=0.0
         )
         self.operations: Optional[EdiOperations] = []
 
+    def is_active(self) -> bool:
+        """Returns True if the workflow instance is active"""
+        return self.current_state not in (EdiOperations.COMPLETE, EdiOperations.FAIL)
```

Comment:
How important is it that we maintain the workflow state, operations and the active concept?  We did not wind up using those in the connect workflows.

Reply:
I can remove this and make the "step" functions "private by convention" (_enrich, _validate, etc) to signal that we don't intend folks to use them directly.